src/Entity/Invoice.php

```diff
@@ -53,6 +53,9 @@ class Invoice implements InvoiceInterface
     /** @var ChannelInterface */
     protected $channel;
 
+    /** @var bool */
+    protected $paymentState;
```

Comment:
Is `$paymentState` a good property name that holds only true and false? I would vote for something like `isPaid` Payment state is more like payment status to me.

Reply:
I also have seen that in the first iteration, you have implemented statuses. Why did you change your mind?